File: AFMobile/AFAndroid/app/src/main/java/cz/cvut/fel/matyapav/afandroid/enums/SupportedComponents.java
package cz.cvut.fel.matyapav.afandroid.enums;

/**
 * Enumeration of supported components
 *
 * @author Pavel Matyáš (matyapav@fel.cvut.cz).
 *@since 1.0.0..
 */
public enum SupportedComponents {

    FORM("Form"),
    TABLE("Table"), //not yet fully supported
    LIST("List");

    SupportedComponents(String type) {
        this.type = type;
    }

    private String type;

    public String getType() {
        return type;
    }

}

File: AFMobile/AFWinPhone/AFWinPhone/enums/SupportedComponents.cs
using System;

namespace AFWinPhone.enums
{
    public sealed class SupportedComponents
    {
        public static readonly SupportedComponents FORM = new SupportedComponents("Form");
        public static readonly SupportedComponents LIST = new SupportedComponents("List");

        private String type;

        SupportedComponents(String type)
        {
            this.type = type;
        }

        public String getType()
        {
            return type;
        }
    }
}
